lessons/07-tests-tasks/inclass/README.md

```diff
@@ -0,0 +1,13 @@
+# Cat App with Testing
+
+## Dependencies
+Before running the app,
+`npm install -g jshint` to globally install jshint, a command-line code quality checking tool
```

Comment:
A possible in-class exercise (or after class) would be to add a jshint plugin to your editor of choice (https://atom.io/packages/linter-jshint)